bgpd/bgp_evpn.c

```diff
@@ -4459,7 +4459,8 @@ void bgp_evpn_configure_import_rt_for_vrf(struct bgp *bgp_vrf,
 					  struct ecommunity *ecomadd)
 {
 	/* uninstall routes from vrf */
-	uninstall_routes_for_vrf(bgp_vrf);
+	if (is_l3vni_live(bgp_vrf))
```

Comment:
Why do we need to control the uninstall of the routes with this check ? Uninstallation should be fine to do in this case. Right ?

Reply:
We shouldn't have any routes installed, so I put this in for symmetry and as an optimization. I don't see any harm in the check.